README.md

```diff
@@ -91,78 +44,42 @@ See "Reference architecture" in Authoring Guidelines in the public documentation
 https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
 -->
 
-## Usage
-
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
-```hcl
-
-```
+<!-- BEGIN OVERVIEW HOOK -->
+## Overview
+* [terraform-ibm-zvsi](#terraform-ibm-zvsi)
+* [Submodules](./)
+    * [quickstart](./solutions/quickstart/)
+    * [standard](./solutions/standard/)
+* [Contributing](#contributing)
+<!-- END OVERVIEW HOOK -->
 
 ## Required IAM access policies
 
-<!-- PERMISSIONS REQUIRED TO RUN MODULE
-If this module requires permissions, uncomment the following block and update
-the sample permissions, following the format.
-Replace the sample Account and IBM Cloud service names and roles with the
-information in the console at
-Manage > Access (IAM) > Access groups > Access policies.
--->
-
-<!--
 You need the following permissions to run this module.
 
 - Account Management
-    - **Sample Account Service** service
-        - `Editor` platform access
-        - `Manager` service access
+    - **Resource Group** service
+        - `Viewer` platform access
     - IAM Services
-        - **Sample Cloud Service** service
-            - `Administrator` platform access
--->
-
-<!-- NO PERMISSIONS FOR MODULE
-If no permissions are required for the module, uncomment the following
-statement instead the previous block.
--->
+            - `Editor` platform access
+        - **VPC Infrastructure Services** service
```

Comment:
Check with Akshay on the complete list of permissions required for this module and update here.

Reply:
Made changes as suggested by Akshay to take the reference from IBM cloud permissions for quickstart and standard variation.